app/pb/api.proto

```diff
@@ -225,34 +267,53 @@ message UpdateProfileReq {
   google.protobuf.StringValue name = 1;
 
   google.protobuf.StringValue city = 2;
+  NullableStringValue hometown = 100;
   // doubles are enough to describe lat/lng
   // in EPSG4326, lat & lon are in degress, check the EPS4326 definition for details
   google.protobuf.DoubleValue lat = 22;
   google.protobuf.DoubleValue lng = 23;
   google.protobuf.DoubleValue radius = 24; // metres
 
   google.protobuf.StringValue gender = 3;
+  NullableStringValue pronouns = 101;
   NullableStringValue occupation = 4;
+  NullableStringValue education = 102;
   NullableStringValue about_me = 5;
+  NullableStringValue my_travels = 103;
+  NullableStringValue things_i_like = 104;
   NullableStringValue about_place = 6;
-  google.protobuf.StringValue color = 7;
 
   HostingStatus hosting_status = 8;
+  MeetupStatus meetup_staus = 105;
 
   RepeatedStringValue languages = 9;
   RepeatedStringValue countries_visited = 10;
   RepeatedStringValue countries_lived = 11;
+  NullableStringValue additional_information = 106;
 
   NullableUInt32Value max_guests = 12;
-  NullableBoolValue multiple_groups = 13;
   NullableBoolValue last_minute = 14;
+  NullableBoolValue has_pets = 107;
   NullableBoolValue accepts_pets = 15;
+  NullableStringValue pet_details = 108;
+  NullableBoolValue has_kids = 109;
   NullableBoolValue accepts_kids = 16;
+  NullableStringValue kid_details = 110;
+  NullableBoolValue has_housemates = 111;
+  NullableStringValue housemate_details = 112;
   NullableBoolValue wheelchair_accessible = 17;
   SmokingLocation smoking_allowed = 18;
-  NullableStringValue sleeping_arrangement = 19;
+  NullableBoolValue smokes_at_home = 113;
+  NullableBoolValue drinks_allowed = 114;
+  NullableBoolValue drinks_at_home = 115;
+  NullableStringValue other_host_info = 116;
+  SleepingArrangement sleeping_arrangement = 117;
+  NullableStringValue sleeping_details = 118;
   NullableStringValue area = 20;
   NullableStringValue house_rules = 21;
+  NullableBoolValue parking = 119;
+  ParkingDetails parking_details = 120;
+  NullableBoolValue Camping_OK = 121;
```

Comment:
```suggestion
  NullableBoolValue camping_ok = 121;
```

Reply:
Most of the places I capitalized it. I see I missed it in a few places. Unless you think it should never be capitalized?